[2024-10-09T10:59:19.201+0700] {processor.py:157} INFO - Started process (PID=3953) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T10:59:19.202+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T10:59:19.204+0700] {logging_mixin.py:149} INFO - [2024-10-09T10:59:19.203+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T10:59:19.257+0700] {logging_mixin.py:149} INFO - [2024-10-09T10:59:19.255+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T10:59:19.258+0700] {logging_mixin.py:149} INFO - [2024-10-09T10:59:19.257+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T10:59:19.259+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T10:59:19.281+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.086 seconds
[2024-10-09T11:01:14.637+0700] {processor.py:157} INFO - Started process (PID=4132) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:01:14.656+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:01:14.657+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:01:14.657+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:01:14.675+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:01:14.673+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:01:14.676+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:01:14.675+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:01:14.676+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:01:14.699+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-10-09T11:02:32.492+0700] {processor.py:157} INFO - Started process (PID=4204) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:02:32.494+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:02:32.495+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:02:32.495+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:02:32.501+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:02:32.499+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:02:32.501+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:02:32.501+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:02:32.502+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:02:32.524+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2024-10-09T11:03:31.160+0700] {processor.py:157} INFO - Started process (PID=4258) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:03:31.199+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:03:31.201+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:03:31.200+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:03:31.209+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:03:31.207+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:03:31.209+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:03:31.209+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:03:31.210+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:03:31.231+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.079 seconds
[2024-10-09T11:04:27.960+0700] {processor.py:157} INFO - Started process (PID=4314) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:04:28.271+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:04:28.274+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:04:28.274+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:04:28.449+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:04:28.447+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:04:28.450+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:04:28.449+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:04:28.450+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:04:28.787+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.834 seconds
[2024-10-09T11:07:22.388+0700] {processor.py:157} INFO - Started process (PID=4613) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:07:22.402+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:07:22.404+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:07:22.404+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:07:22.455+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:07:22.411+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:07:22.456+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:07:22.455+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:07:22.457+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:07:22.480+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.099 seconds
[2024-10-09T11:08:26.468+0700] {processor.py:157} INFO - Started process (PID=4682) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:08:26.495+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:08:26.498+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:08:26.497+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:08:26.550+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:08:26.537+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:08:26.551+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:08:26.551+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:08:26.552+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:08:26.577+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.116 seconds
[2024-10-09T11:09:23.259+0700] {processor.py:157} INFO - Started process (PID=4737) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:09:23.272+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:09:23.274+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:09:23.274+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:09:23.282+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:09:23.281+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:09:23.283+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:09:23.283+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:09:23.283+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:09:23.306+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.053 seconds
[2024-10-09T11:10:33.637+0700] {processor.py:157} INFO - Started process (PID=4815) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:10:33.657+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:10:33.660+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:10:33.659+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:10:33.668+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:10:33.666+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:10:33.668+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:10:33.668+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:10:33.669+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:10:33.689+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.059 seconds
[2024-10-09T11:11:28.235+0700] {processor.py:157} INFO - Started process (PID=4872) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:11:28.250+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:11:28.253+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:11:28.252+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:11:28.280+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:11:28.277+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:11:28.281+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:11:28.281+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:11:28.282+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:11:28.303+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.074 seconds
[2024-10-09T11:12:21.061+0700] {processor.py:157} INFO - Started process (PID=4930) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:12:21.103+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:12:21.105+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:12:21.105+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:12:21.114+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:12:21.113+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:12:21.115+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:12:21.115+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:12:21.115+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:12:21.136+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.082 seconds
[2024-10-09T11:13:14.222+0700] {processor.py:157} INFO - Started process (PID=4982) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:13:14.236+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:13:14.238+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:13:14.238+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:13:14.247+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:13:14.245+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:13:14.248+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:13:14.247+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:13:14.249+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:13:14.270+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.055 seconds
[2024-10-09T11:14:07.024+0700] {processor.py:157} INFO - Started process (PID=5038) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:14:07.056+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:14:07.058+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:14:07.057+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:14:07.066+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:14:07.065+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:14:07.067+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:14:07.066+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:14:07.067+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:14:07.088+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.070 seconds
[2024-10-09T11:16:10.350+0700] {processor.py:157} INFO - Started process (PID=5305) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:16:10.364+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:16:10.366+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:16:10.366+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:16:10.393+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:16:10.372+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:16:10.394+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:16:10.394+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:16:10.395+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:16:10.436+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.099 seconds
[2024-10-09T11:17:19.618+0700] {processor.py:157} INFO - Started process (PID=5376) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:17:19.648+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:17:19.650+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:17:19.650+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:17:19.661+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:17:19.659+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:17:19.665+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:17:19.663+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:17:19.668+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:17:19.705+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.099 seconds
[2024-10-09T11:18:28.682+0700] {processor.py:157} INFO - Started process (PID=5442) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:18:28.724+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:18:28.725+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:18:28.725+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:18:28.758+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:18:28.757+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:18:28.759+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:18:28.759+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:18:28.760+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:18:28.786+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.111 seconds
[2024-10-09T11:19:31.240+0700] {processor.py:157} INFO - Started process (PID=5512) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:19:31.254+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:19:31.256+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:19:31.255+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:19:31.262+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:19:31.261+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:19:31.268+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:19:31.263+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:19:31.269+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:19:31.297+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-10-09T11:20:36.392+0700] {processor.py:157} INFO - Started process (PID=5590) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:20:36.434+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:20:36.436+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:20:36.435+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:20:36.449+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:20:36.447+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:20:36.450+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:20:36.449+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:20:36.450+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:20:36.544+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.161 seconds
[2024-10-09T11:21:44.879+0700] {processor.py:157} INFO - Started process (PID=5652) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:21:44.891+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:21:44.893+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:21:44.892+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:21:44.905+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:21:44.903+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:21:44.908+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:21:44.905+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:21:44.909+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:21:44.954+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.081 seconds
[2024-10-09T11:22:46.399+0700] {processor.py:157} INFO - Started process (PID=5714) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:22:46.438+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:22:46.440+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:22:46.440+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:22:46.451+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:22:46.446+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:22:46.451+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:22:46.451+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:22:46.452+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:22:46.485+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.098 seconds
[2024-10-09T11:23:53.879+0700] {processor.py:157} INFO - Started process (PID=5782) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:23:53.919+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:23:53.921+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:23:53.921+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:23:53.934+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:23:53.933+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:23:53.939+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:23:53.939+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:23:53.940+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:23:53.979+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.112 seconds
[2024-10-09T11:25:00.643+0700] {processor.py:157} INFO - Started process (PID=5849) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:25:00.650+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:25:00.652+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:25:00.651+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:25:00.666+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:25:00.664+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:25:00.667+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:25:00.667+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:25:00.668+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:25:00.702+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.071 seconds
[2024-10-09T11:28:27.574+0700] {processor.py:157} INFO - Started process (PID=6008) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:28:27.599+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:28:27.603+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:28:27.601+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:28:27.621+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:28:27.619+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:28:27.622+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:28:27.621+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:28:27.623+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:28:27.654+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.088 seconds
[2024-10-09T11:31:23.345+0700] {processor.py:157} INFO - Started process (PID=6141) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:31:25.065+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:31:25.067+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:31:25.067+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:31:25.643+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:31:25.497+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:31:25.645+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:31:25.644+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:31:25.711+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:31:28.918+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 5.664 seconds
[2024-10-09T11:34:49.998+0700] {processor.py:157} INFO - Started process (PID=6548) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:34:50.212+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:34:50.215+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:34:50.214+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:34:50.328+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:34:50.230+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:34:50.329+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:34:50.329+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:34:50.330+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:34:50.354+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.363 seconds
[2024-10-09T11:37:58.374+0700] {processor.py:157} INFO - Started process (PID=6810) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:37:58.431+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-09T11:37:58.459+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:37:58.458+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:37:59.041+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:37:58.796+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-09T11:37:59.042+0700] {logging_mixin.py:149} INFO - [2024-10-09T11:37:59.041+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-09T11:37:59.067+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-09T11:37:59.121+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.758 seconds
